src/vl-radio-group.js

```diff
@@ -0,0 +1,26 @@
+import { VlElement, define } from '/node_modules/vl-ui-core/dist/vl-core.js';
+
+/**
+ * VlRadioGroup
+ * @class
+ * @classdesc De radio group bundelt verschillende radio elementen die samenhoren.
+ *
+ * @extends VlElement
+ * 
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-radio/releases/latest|Release notes}
+ * @see {@link https://www.github.com/milieuinfo/webcomponent-vl-ui-radio/issues|Issues}
+ * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-radio-group.html|Demo}
+ */
+export class VlRadioGroup extends VlElement(HTMLElement) {
+  connectedCallback() {
+    this.radios.forEach((radio) => {
+      radio.setAttribute('data-vl-name', 'radio');
+    });
+  }
```

Comment:
Volgens mij moet de naam echt random zijn per group:

Als je 2 groepen maakt met elk verschillende radiobuttons heb, kan je niet 1 radiobutton per groep selecteren volgens mij.

Stel je hebt 2 groepen met elk 2 radiobuttons, kan je er maar 1 van de 4 selecteren ipv 2 keer 1 van de 2:

`<vl-radio-group>
        <vl-radio id="radio-1" data-vl-label="Ja" data-vl-value="yes"></vl-radio>
        <vl-radio id="radio-2" data-vl-label="Nee" data-vl-value="no"></vl-radio>
      </vl-radio-group>
      <vl-radio-group>
        <vl-radio id="radio-3" data-vl-label="Ja1" data-vl-value="yes2"></vl-radio>
        <vl-radio id="radio-4" data-vl-label="Nee2" data-vl-value="no2"></vl-radio>
      </vl-radio-group>`

Reply:
Je hebt gelijk, de vl-ui-radio-group heeft (momenteel) geen shadow DOM, dus moet het name attribuut uniek zijn of moeten we de vl-ui-radio-group shadow DOM geven.

@SanderKleykens @Pieter-Beckers @pascalriquier en @wautergu voorkeuren?

Reply:
ah ik ging er van uit dat die shadow dom had omdat die van VlElement extende. Ik ben voor shadow dom precies.

Reply:
met shadow dom ben je wellicht zekerder dan met iets 'random'?

Reply:
De implementatie die ik had voor iets random was redelijk random, dus de kans dat je dubbels hebt op één pagina lijken mij zeer klein. Shadow DOM introduceren is mogelijk.

Reply:
Ik ging er inderdaad van uit dat de vl-ui-radio-group een shadow dom had.